src/Components/ListCards.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import TouristInfoCards from "./TouristInfoCards";
+
+const ListCards = () => {
+  return (
+    <div className="list-cards">
+      <TouristInfoCards
+        imgUrl="https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcTHS56YNuaaY7IXR7iOa2ptlsAXaBNN0dvr-bEXgfxdsaVbmkYn&usqp=CAU"
+        link="https://peoplemakeglasgow.com"
+        name="Glasgow"
+      />
+      <TouristInfoCards
+        imgUrl="https://i2-prod.manchestereveningnews.co.uk/incoming/article17428679.ece/ALTERNATES/s615/1_MWOCT19-121051JPG.jpg"
+        link="https://visitmanchester.com"
+        name="Manchester"
+      />
+      <TouristInfoCards
+        imgUrl="https://d36tnp772eyphs.cloudfront.net/blogs/1/2011/05/London-England-Big-Ben-Cityscape-destinations-1200x900.jpg"
+        link="https://visitlondon.com"
+        name="London"
+      />
+    </div>
```

Comment:
Good job. I like the way you created this component and is reusing it. What if we created a data structure which we could map and return this component? Wouldn't be cool? We could avoid copy and paste this component over and over again.

Something like:
```js
const touristInformations = [{
   city: 'Glasgow',
   site: 'https://peoplemakeglasgow.com/',
   imageURL: '"https://frontline-collections.com/wp-content/uploads/2018/02/Glasgow-Debt-Collection.jpg'
}, {
   city: 'Manchester',
   site: 'https://www.visitmanchester.com/',
   imageURL: 'https://www.prolificnorth.co.uk/sites/default/files/styles/lightbox_large/public/images/news/manchester-cityscape0.jpg?itok=VjQDdTYJ'
}];
```

What do you think?

Reply:
Yes. It's nicer!
`const cityUk = [
    {
      imgUrl:
        "https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcTHS56YNuaaY7IXR7iOa2ptlsAXaBNN0dvr-bEXgfxdsaVbmkYn&usqp=CAU",
      link: "https://peoplemakeglasgow.com",
      name: "Glasgow",
    },
    {
      imgUrl:
        "https://i2-prod.manchestereveningnews.co.uk/incoming/article17428679.ece/ALTERNATES/s615/1_MWOCT19-121051JPG.jpg",
      link: "https://visitmanchester.com",
      name: "Manchester",
    },
    {
      imgUrl:
        "https://d36tnp772eyphs.cloudfront.net/blogs/1/2011/05/London-England-Big-Ben-Cityscape-destinations-1200x900.jpg",
      link: "https://visitlondon.com",
      name: "London",
    },
  ];
  return (
    <div className="list-cards">
      {cityUk.map((city) => (
        <TouristInfoCards {...city} />
      ))}
    </div>
  );
};`

Reply:
Yes! It's nicer!
`const cityUk = [
    {
      imgUrl:
        "https://encrypted-tbn0.gstatic.com/images?q=tbn%3AANd9GcTHS56YNuaaY7IXR7iOa2ptlsAXaBNN0dvr-bEXgfxdsaVbmkYn&usqp=CAU",
      link: "https://peoplemakeglasgow.com",
      name: "Glasgow",
    },
    {
      imgUrl:
        "https://i2-prod.manchestereveningnews.co.uk/incoming/article17428679.ece/ALTERNATES/s615/1_MWOCT19-121051JPG.jpg",
      link: "https://visitmanchester.com",
      name: "Manchester",
    },
    {
      imgUrl:
        "https://d36tnp772eyphs.cloudfront.net/blogs/1/2011/05/London-England-Big-Ben-Cityscape-destinations-1200x900.jpg",
      link: "https://visitlondon.com",
      name: "London",
    },
  ];
  return (
    <div className="list-cards">
      {cityUk.map((city) => (
        <TouristInfoCards {...city} />
      ))}
    </div>
  );
};`